meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-tpm2

```diff
@@ -70,27 +70,39 @@ EOF
             if [ -n "${override}" ]; then
                     echo "${override}" | _hexdecode | tcgtool "$var" | _sha256
             else
-                    tcgtool "$var" | _sha256
+                    dd if="${var}" \
```

Comment:
This could also use a comment saying that we do not know why, but it does not work without `dd`